api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md

```diff
@@ -22,25 +22,25 @@ Indicates if the dragged row is dropped inside another row. Available only for t
 #include common-ref-eventparam
 
 ##### field(e.fromComponent): dxSortable | dxDraggable
-The instance of the row's source UI component.
+The instance of the row's source UI component. Works only for dragging between components.
 
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the source UI component. Works only for dragging between components.
```

Comment:
We should avoid the -ing form if possible. Can you clarify the sentence so that it specifies what is being dragged between components?

Reply:
To avoid -ing, I can rewrite the sentence in the following way: "Works only for drag and drop elements between components. In the following demo you can see and try how drag rows between 2 components: https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/DnDBetweenGrids/

Reply:
To avoid -ing, I can rewrite the sentence in the following way: "Works only for drag and drop elements between components. In the following demo you can see and try how drag rows between 2 components: https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/DnDBetweenGrids/

Reply:
I would rephrase it as:  
Works only for element drag and drop between components. In the following demo you can see how to drag rows between two components: https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/DnDBetweenGrids/.

If you're okay with this, can you replace all instances of this throughout the PR?

Reply:
I specified second sentence about demo only for you) In the table of options I cannot specify this sentence.

Reply:
Oh okay, I get it. Thanks.